src/sets.jl

```diff
@@ -30,6 +30,29 @@ julia> dimension(PositiveSemidefiniteConeTriangle(2))
 """
 function dimension end
 
+"""
+    dual_set(s::AbstractSet)
+
+Return the dual set of `s`, in terms of the dual cone of the set. 
+See https://en.wikipedia.org/wiki/Dual_cone_and_polar_cone for more information.
+
```

Comment:
The discussion about what the function returns if the dual set is not defined got lost somehow.